ConsoleApps/General-ConsoleApp/General-ConsoleApp/General-ConsoleApp.csproj

```
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <RootNamespace>General_ConsoleApp</RootNamespace>
```

I would actually rethink the name (General-ConsoleApp) doesn't say much, right? 

```suggestion
    <RootNamespace>General.ConsoleApp</RootNamespace>
```